src/org/labkey/test/util/AuditLogHelper.java

```diff
@@ -69,6 +76,116 @@ public DataRegionTable goToAuditEventView(String eventType)
         return new DataRegionTable("query", _wrapper);
     }
 
+    /**
+     * Get the audit logs from the LabKey server filtered to the given project.
+     *
+     * @param projectName Name of the LK project to filter to.
+     * @param folderName Name of the LK folder to filter to.
+     * @param auditEventName Name of the audit event to filter on. Example 'SamplesWorkflowAuditEvent'.
+     * @param columnNames The name of the columns to return.
+     * @param filters The filters to be applied
+     * @return A rowResponse with the query logs.
+     * @throws IOException Can be thrown by the SelectRowsCommand.
+     * @throws CommandException Can be thrown by the SelectRowsCommand.
+     */
+    public static SelectRowsResponse getAuditLogsFromLKS(String projectName, String folderName, String auditEventName, List<String> columnNames,
+                                                         List<Filter> filters, @Nullable Integer maxRows) throws IOException, CommandException
+    {
```

Comment:
This should just take the containerPath as a single parameter. Having separate projectName and folderName is confusing when they are equal; technically, you could have a subfolder with the same name as the project.
```suggestion
    public static SelectRowsResponse getAuditLogsFromLKS(String containerPath, String auditEventName, List<String> columnNames,
                                                         List<Filter> filters, @Nullable Integer maxRows) throws IOException, CommandException
    {
        String projectName = StringUtils.strip(containerPath, " /").split("/")[0];
```

Reply:
done